pilot/pkg/serviceregistry/kube/controller/endpoint_builder.go

```diff
@@ -38,7 +39,8 @@ func NewEndpointBuilder(c *Controller, pod *v1.Pod) *EndpointBuilder {
 	var podLabels labels.Instance
 	if pod != nil {
 		locality = c.getPodLocality(pod)
-		sa = kube.SecureNamingSAN(pod)
+		td := spiffe.GetTrustDomainByCluster(c.clusterID)
```

Comment:
What does it mean ? The trust domains are mesh wide ( or at least tenant-wide ), they are not specific to a cluster.

Reply:
obsolete.